extensions/amp-form/0.1/test/test-amp-form.js

```diff
@@ -678,7 +680,9 @@ describes.repeated('', {
 
           const xhrCall = ampForm.xhr_.fetch.getCall(0);
           const config = xhrCall.args[1];
-          expect(config.body).to.not.be.null;
+          expect(config.body).to.be.an.instanceof(FormDataWrapper);
+          expect(fromIterator(config.body.entries())).to.have.deep.members(
+              fromIterator(new FormDataWrapper(getForm()).entries()));
```

Comment:
Nit: This line is pretty dense. Can you unpack with a local var?

Reply:
Done.